spec/defines/vhost_spec.rb

```diff
@@ -154,6 +154,7 @@
           'ssl_verify_depth'            => '3',
           'ssl_options'                 => '+ExportCertData',
           'ssl_openssl_conf_cmd'        => 'DHParameters "foo.pem"',
+          'ssl_proxy_verify'            => 'require',
```

Comment:
Since the template checks for existence of `@ssl_proxy_verify` before adding the line to the config, should this default be to `undef`? By setting a default, this means that the `SSLProxyVerify` line will always be present in the default config. Is that what we want?

cc/ @igalic